drivers/i2c/Kconfig.sam_twihs

```diff
@@ -7,5 +7,6 @@ config I2C_SAM_TWIHS
 	bool "Atmel SAM (TWIHS) I2C driver"
 	default y
 	depends on DT_HAS_ATMEL_SAM_I2C_TWIHS_ENABLED
+	select HAS_I2C_RTIO
```

Comment:
Do we need this as well for `I2C_MCUX_LPI2C`?

``` code
config I2C_MCUX_LPI2C
	bool "MCUX LPI2C driver"
	default y
	select HAS_I2C_RTIO
	...
```

Reply:
Yes we do, thank you